retrieveProducts/scripts/ddl.sql

```diff
@@ -0,0 +1,19 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE product (
+	id uuid DEFAULT uuid_generate_v4() NOT NULL,
+	title VARCHAR(255) NOT NULL,
+	description VARCHAR(1000) NOT NULL,
+	price INTEGER DEFAULT 0,
+	CONSTRAINT pk_product PRIMARY KEY(id)
+);
+
+
+CREATE TABLE store ( 
+	product_id uuid DEFAULT uuid_generate_v4() NOT NULL,
```

Comment:
I don't think it makes sense to set default value for FK column

Reply:
Agree will fix this

Reply:
done